app/assets/javascripts/organizations.js

```diff
@@ -60,3 +66,36 @@ function displayMap(address) {
     }
   });
 }
+
+//This makes the description field autosize to the size of the text content
+var observe;
+if (window.attachEvent) {
+  observe = function (element, event, handler) {
+    element.attachEvent('on'+event, handler);
+  };
+}
+else {
+  observe = function (element, event, handler) {
+    element.addEventListener(event, handler, false);
+  };
+}
+function initTextArea () {
+  var text = document.getElementById('organization_description');
+  function resize () {
+    text.style.height = 'auto';
+    text.style.height = text.scrollHeight+'px';
+  }
+  /* 0-timeout to get the already changed text */
+  function delayedResize () {
+    window.setTimeout(resize, 0);
+  }
+  observe(text, 'change',  resize);
+  observe(text, 'cut',     delayedResize);
+  observe(text, 'paste',   delayedResize);
+  observe(text, 'drop',    delayedResize);
+  observe(text, 'keydown', delayedResize);
```

Comment:
Can we use jQuery to observe and fire the events? What are the benefits of using the custom solution here?